File: InsuranceApp/ContractService/Models/Contract.cs
namespace ContractService.Models
{
    public class Contract
    {
        public int Id { get; set; }
        public string FullName { get; set; }
        public string Registration { get; set; }
        public string Address { get; set; }
        public string CarMake { get; set; }
        public string CarModel { get; set; }
        public int Year { get; set; }
        public bool Tief { get; set; }
        public bool Weather { get; set; }
        public bool Crash { get; set; }
        public int Mounth { get; set; }
        public DateTime CreatedOn { get; set; }
        public bool Activated { get; set; }

    }
}

File: Frontend/src/app/models/Contract.ts
export class Contract {
    id:number;
    fullName: string;
    registration: string;
    address: string;
    carMake: string = "";
    carModel: string = "";
    year: number = new Date().getFullYear();
    tief: boolean = false;
    tiefText: string = "";
    weather: boolean = false;
    weatherText: string = "";
    crash: boolean = false;
    crashText: string = "";
    mounth: number = 1;
    createdOn: Date = new Date();
    activated: boolean = false;

    constructor(id: number = 0, fullName: string = "", registration: string = "", address: string = "") {
        this.id = id;
        this.fullName = fullName;
        this.registration = registration;
        this.address = address;
    }
}